meerk40t/core/elements/element_treeops.py

```diff
@@ -167,24 +167,49 @@ def image_convert_modifier(node, **kwargs):
             type="elem image",
         )
 
+    @tree_operation(_("Remove effect"), node_type=effect_nodes, help="")
+    def remove_effect(node, **kwargs):
+        childs = [e for e in node._children]
+        for e in childs:
+            e._parent = None  # Otherwise add_node will fail below
+            node.parent.add_node(e)
```

Comment:
This is actually `append_child` it might need a bit of a better name, but append much like `insert_sibling` detaches the child, and calls the relevant notifications.

It's probably fine, but we're sort of supposed to call the notifications for detaching and attaching.